src/DIRAC/WorkloadManagementSystem/Client/Limiter.py

```diff
@@ -12,10 +21,109 @@
 from DIRAC.WorkloadManagementSystem.Client import JobStatus
 
 
+class TwoLevelCache:
```

Comment:
It does not seem to me there's anything "specific" to this class. What if you move it to a generic utility module?